samples/bluetooth/mesh/light_ctrl/prj.conf

```diff
@@ -21,6 +21,7 @@ CONFIG_PWM=y
 CONFIG_PM_SINGLE_IMAGE=y
 CONFIG_PM_PARTITION_SIZE_SETTINGS_STORAGE=0x8000
 CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=y
+CONFIG_FPU=y
```

Comment:
Probably the issue is that the regulator defaults to `y` if FPU option is enabled:
https://github.com/nrfconnect/sdk-nrf/blob/cf827943561d12b86efd8f78c828c254bc5186ee/subsys/bluetooth/mesh/Kconfig.models#L287-L289
but this seems confusing to me. Perhaps we could change regulator Kconfig option to select FPU and then explicitly enable the regulator in the sample.

Reply:
Also, I think, `menuconfig BT_MESH_LIGHT_CTRL_REG` should be moved inside `if BT_MESH_LIGHT_CTRL_SRV`

Reply:
> Also, I think, `menuconfig BT_MESH_LIGHT_CTRL_REG` should be moved inside `if BT_MESH_LIGHT_CTRL_SRV`

This I'm not sure. IIRC, the idea was that regulator could be used outside of light lc model.